simpleeval.py

```diff
@@ -526,24 +526,30 @@ def _eval_name(self, node):
         try:
             # This happens at least for slicing
             # This is a safe thing to do because it is impossible
-            # that there is a true exression assigning to none
+            # that there is a true expression assigning to none
```

Comment:
Would you be open to me running the codebase through [codespell](https://github.com/codespell-project/codespell) and [typos](https://github.com/crate-ci/typos)?

Reply:
Yes indeed!